===================================================================
RCS file: /home/cvsroot/afterstep-stable/libAfterImage/asvisual.c,v
rcsdiff: /home/cvsroot/afterstep-stable/libAfterImage/asvisual.c,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.111
retrieving revision 1.112
diff -u -r1.111 -r1.112
--- afterstep-stable/libAfterImage/asvisual.c	2008/05/29 22:23:25	1.111
+++ afterstep-stable/libAfterImage/asvisual.c	2008/11/12 22:38:31	1.112
@@ -404,9 +404,9 @@
 		}
 #ifdef HAVE_GLX
 		if( asv->glx_scratch_gc_direct )
-			glXDestroyContext(dpy, asv->glx_scratch_gc_direct );
+			glXDestroyContext(asv->dpy, asv->glx_scratch_gc_direct );
 		if( asv->glx_scratch_gc_indirect )
-			glXDestroyContext(dpy, asv->glx_scratch_gc_indirect );
+			glXDestroyContext(asv->dpy, asv->glx_scratch_gc_indirect );
 #endif
 		if( asv->scratch_window ) 
 			XDestroyWindow( asv->dpy, asv->scratch_window );
